locale/en.json

```diff
@@ -39,6 +39,7 @@
     "copy": "Copy"
   },
   "sidebar": {
+    "nodeFilter": "Node filter",
```

Comment:
i disagree with @SimJoSt, because the list says new nodes, missing nodes or tab Nodes. Not a single term with "Host" or "Hostname"